Projects/CSS/01_Card/README.md

```diff
@@ -0,0 +1,31 @@
+# Course Listing Page
+<p>This is a static webpage project built using HTML and CSS to showcase different course offerings from CodeHelp. Each course is displayed in a card format with information such as course title, discounted price, original price, and discount percentage. The project utilizes custom CSS for layout and Google Fonts for typography.</p>
+
+# Project Structure
+The project contains the following files and folders:</br>
+.</br>
+├── index.html          # Main HTML file that contains the structure of the webpage</br>
+├── css/</br>
+│   └── card.css        # External CSS file for styling the cards and layout</br>
+├── images/</br>
+│   ├── thumbnail-1.jpg # Thumbnail image for the Web Development course</br>
+│   ├── thumbnail-2.jpg # Thumbnail image for the DSA course</br>
+│   └── thumbnail-3.jpg # Thumbnail image for the OOPs course</br>
+└── README.md           # Documentation file (you're reading it!)</br>
+# Features
+* Card-Based Layout: The webpage displays each course in a visually appealing card, including an image, title, price, and discount information.
+* Responsive Design: The webpage is built with responsive design principles in mind, adjusting the layout for different screen sizes.
+* Dynamic Links: Each course card links to an external page for the respective course using the target="_blank" attribute, which opens the link in a new tab.
+* Google Fonts Integration: Custom fonts are used via Google Fonts to enhance the typography of the webpage.
+Technology Stack
+* HTML5: For structuring the content of the webpage.
+* CSS3: Custom CSS is used to design and style the card layout and other elements.
+* Google Fonts: The Rubik font is imported from Google Fonts for a clean, modern look.
+* Static Assets: Images are used for course thumbnails, stored in the images/ directory.
+# How to Run the Project
+* Download or clone this repository: 
+* git clone https://github.com/your-username/codehelp-course-listing.git
+* Open the index.html file in your browser:
```

Comment:
Make bullet points, something like this

1. Download or clone this repository:
    `git clone https://github.com/your-username/codehelp-course-listing.git`

2. Open the index.html file in your browser:
    - Navigate to the project directory.
    - Open index.html in any web browser to view the course listing page.

Reply:
![image](https://github.com/user-attachments/assets/86c39a27-62d7-4f1d-ab61-a9d89fa76a8f)
Done